docs/source/tutorials/extending_mantid_with_python/python_fit_functions/03_attributes.rst

```diff
@@ -45,3 +45,25 @@ If defined in your class then you can use it to set a python attribute on your o
             if name == "NLoops":
                 # Can the be accessed quicker later using self._nloops
                 self._nloops = value
+
+If desired, the possible values that an attribute can take can be restricted through the use
+of a Function Attribute Validator. Such a validator must derive from the ``IValidator``
+abstract class; a number of these are currently provided in the Manitd Kernel such as:
```

Comment:
There is a typo in this line of 'Mantid'

Reply:
Ammended.